### ISSUE TITLE
Handle bash and python code directly

### ISSUE THREAD

**Issue by frikky** [frikky/Shuffle-apps#10]
Make it possible to write scripts (bash/python) directly

**Comment by n3x77** [comment]
As discussed within twitter, maybe we could port https://github.com/nsacyber/WALKOFF-Apps/tree/master/LinuxShell. This would allow execution of shell commands locally and over ssh.

**Comment by frikky** [comment]
Good idea! We should probably do that for more than just shell commands as the apps in theory should work directly in Shuffle.

**Comment by n3x77** [comment]
Could you please describe in more detail what this means? Cause I'm currently trying to port the app into shuffle.

**Comment by frikky** [comment]
Their new app format is the same format used for Shuffle, but I checked it again and saw that this is their old apps.

I'd suggest translating these ones instead, as they're the new format that uses Docker: https://github.com/nsacyber/WALKOFF/tree/master/apps/ssh/1.0.0

I guess this also means it's about time I write documentation for app creation :woman_dancing:

**Comment by n3x77** [comment]
> Their new app format is the same format used for Shuffle, but I checked it again and saw that this is their old apps.
> 
> I'd suggest translating these ones instead, as they're the new format that uses Docker: https://github.com/nsacyber/WALKOFF/tree/master/apps/ssh/1.0.0
> 
> I guess this also means it's about time I write documentation for app creation 💃

Thanks for pointing that out I will try to use this application. When I saw the WALKOFF app for the first time it seemed to be in an older format. But then I don't have to lose energy in rewriting to the new format.

**Comment by frikky** [comment]
Hey @n3x77, have you gotten any farther? I'll be working on app docs this weekend (finally have time!) :)

**Comment by n3x77** [comment]
Hi Frikky,

Sorry I couldn't find the time for it. I did see testing but think you're
documentation will help a lot.

Frikky <notifications@github.com> schrieb am Do., 30. Juli 2020, 07:41:

**Comment by frikky** [comment]
Bash works! https://twitter.com/shuffleio/status/1304037645006168066?s=20

**Comment by frikky** [comment]
Now for python...

**Comment by frikky** [comment]
This has all been working for years ((:
